.coderabbit.yaml

```
@@ -12,12 +12,23 @@ reviews:
   high_level_summary: true
   
+  # Enable label suggestions and auto-apply
+  suggested_labels: true
+  auto_apply_labels: true
+  
+  # Custom labeling instructions
+  labeling_instructions:
+    - "Apply 'Review Approved' label if the PR has no more suggested coderabbit changes"
+    - "Apply 'CR Review Pending' label if a review was submitted to coderabbit but hasn't been completed"
+    - "Apply 'CR Review Failed' label if changes are requested by coderabbit"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)